### PR TITLE
fixed dependecies

### PR DESCRIPTION
**Pull request by RUGMJ**
credit to @PatrickMSM from the fabric dev server for his help

### CODE REVIEW

**Comment by Matthias1590** [comment]
I saw you post about a call to MinecraftClient.getInstance failing, is that also fixed with this PR?

File: build.gradle [hunk 1]
```diff
@@ -24,13 +24,17 @@ repositories {
 
 dependencies {
 	// jUnit 5
-	implementation 'org.junit.jupiter:junit-jupiter:5.8.1'
+	include implementation ('org.junit.jupiter:junit-jupiter:5.8.1')
```
Comment (Matthias1590) [inline build.gradle:27]: `include implementation` can be replaced with `api` iirc

**Review by Matthias1590 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (RUGMJ) [reply build.gradle:27]: from what ik thats only for mods which are apis, these are regular maven libraries

**Comment by RUGMJ** [comment]
> I saw you post about a call to MinecraftClient.getInstance failing, is that also fixed with this PR?

this fixes that yes

File: build.gradle (hunk 1 above)
Comment (Matthias1590) [reply build.gradle:27]: "api" is an alias for "include implementation"

Comment (RUGMJ) [reply build.gradle:27]: I tried that but it results in the crash that was occurring before

Comment (Matthias1590) [reply build.gradle:27]: I guess this is fine for now though

Comment (Matthias1590) [reply build.gradle:27]: I probably mixed some things up then, good to know that that's fixed now though

**Review by Matthias1590 — APPROVED** [review]

**Pull request merged**